rollup.config.js

```diff
@@ -48,14 +47,19 @@ export default {
       }),
       eslint(),
       svelte({
-        extensions: ['.html', '.svelte', '.svg'],
+        extensions: ['.html', '.svelte'],
         preprocess: preprocessChain,
         dev,
         hydratable: true,
       }),
+      // TODO: figure out the new image caching mechanism
```

Comment:
What exactly about it is to be figured out?

Reply:
Well, do we implement it or not. I think we do, I'll add some more commits in a bit

Reply:
Nah, screw it